kernel/environ.mli

```diff
@@ -113,6 +113,10 @@ val push_rel         : Constr.rel_declaration -> env -> env
 val push_rel_context : Constr.rel_context -> env -> env
 val push_rec_types   : rec_declaration -> env -> env
 
+val push_rel_context_val : Constr.rel_declaration -> rel_context_val -> rel_context_val
```

Comment:
Nope. Don't do that, it allows to break the invariants.

Reply:
As mentioned offline there are no invariants that are currently enforced yet broken by this change.

Reply:
(the only invariants currently enforce are that the env_rel_map and env_rel_ctx in the rel_context_val match, and that the env_nb_rel in the env is the length of the env_rel_context)